python/ray/tests/conftest.py

```diff
@@ -461,6 +461,14 @@ def use_tls(request):
     "type": "smart_open",
     "params": {"uri": f"s3://{bucket_name}/"},
 }
+buffer_open_object_spilling_config = {
```

Comment:
Is it enough to just add these configs? Do we need to refer to them in some tests?

Reply:
Yes, but other s3 tests are pending developers creating a mock bucket for S3, these should be referred to once those are added.